app/src/main/java/com/sun/englishlearning/screen/savedwords/SavedWordsActivity.kt

```diff
@@ -51,6 +52,11 @@ class SavedWordsActivity : BaseActivity<ActivitySavedWordsBinding>() {
         binding.btnBack.setOnClickListener {
             finish()
         }
+
+        binding.etSearch.setOnClickListener {
```

Comment:
```suggestion
        binding.etSearch.setOnClickListener {
```
sao editext lại setOnClick e nhỉ?